client/src/components/scorecard_page/ActionSection.vue

```diff
@@ -62,13 +62,9 @@ export default defineComponent({
 @import 'src/assets/styles/global';
 @import '@blueconduit/copper/scss/01_settings/design-tokens';
 
-.center-container {
-  gap: $spacing-sm;
-  padding: $spacing-lg;
-}
-
-.explain-text {
-  padding: $spacing-md;
+.is-flex {
+  flex-direction: column;
```

Comment:
I tried using [is-flex-direction-column](https://bulma.io/documentation/helpers/flexbox-helpers/) and it didn't work 



<br class="Apple-interchange-newline">